app/ModelSerializers/Summit/Registration/PromoCodes/SummitRegistrationPromoCodeSerializer.php

```diff
@@ -81,6 +82,14 @@ public function serialize($expand = null, array $fields = [], array $relations =
             }
             $values['tags'] = $tags;
         }
+
+        if(in_array('tickets', $relations) && !isset($values['tickets'])) {
```

Comment:
dont add this could be a long list we will create an endpoint if we need this

Reply:
done